t/ui/27-plugin_obs_rsync_status_details.t

```diff
@@ -139,45 +152,49 @@ foreach my $proj (sort { $b cmp $a } keys %params) {
+    like($status, qr/$repo/, "$proj dirty has repo");
 
     # now request fetching builds from obs
-    $driver->find_element("tr#folder_$proj .obsbuildsupdate")->click();
-    my $obsbuilds = _wait_helper("tr#folder_$proj .obsbuilds", sub { shift });
-    is($obsbuilds, $builds_text, "$proj obs builds");
-
-    # now we call forget_run_last() and refresh_last_run() and check once again corresponding columns
-    $driver->find_element("tr#folder_$proj .lastsyncforget")->click();
-    $driver->accept_alert;
-
-    my $lastsync = _wait_helper("tr#folder_$proj .lastsync", sub { !shift });
-    unlike($lastsync, qr/$dt/, "$proj last sync forgotten");
-    is($lastsync, '', "$proj last sync is empty");
-
-    # refresh page and make sure that last sync is gone
-    $driver->get("/admin/obs_rsync/$parent");
-    is($driver->find_element("tr#folder_$proj .lastsync")->get_text(), 'no data', "$proj last sync absent from web UI");
-
+    $driver->find_element("tr#folder_$ident .obsbuildsupdate")->click();
+    my $obsbuilds = _wait_helper("tr#folder_$ident .obsbuilds", sub { shift });
+    is($obsbuilds, ($builds_text ? $builds_text : 'No data'), "$proj obs builds");
+
+    if ($dt ne 'no data') {
+        # now we call forget_run_last() and refresh_last_run() and check once again corresponding columns
+        $driver->find_element("tr#folder_$ident .lastsyncforget")->click();
+        $driver->accept_alert;
+
+        my $lastsync = _wait_helper("tr#folder_$ident .lastsync", sub { !shift });
+        unlike($lastsync, qr/$dt/, "$proj last sync forgotten");
+        is($lastsync, '', "$proj last sync is empty");
+
+        # refresh page and make sure that last sync is gone
+        $driver->get("/admin/obs_rsync/$parent");
+        is($driver->find_element("tr#folder_$ident .lastsync")->get_text(),
+            'no data', "$proj last sync absent from web UI");
+    }
     # Update project status
-    $driver->find_element("tr#folder_$proj .dirtystatusupdate")->click();
+    $driver->find_element("tr#folder_$ident .dirtystatusupdate")->click();
     # now wait until gru picks the task up
     my $dirty_status
-      = _wait_helper("tr#folder_$proj .dirtystatuscol .dirtystatus", sub { index(shift, 'dirty') == -1 });
+      = _wait_helper("tr#folder_$ident .dirtystatuscol .dirtystatus", sub { index(shift, 'dirty') == -1 });
 
     unlike($dirty_status, qr/dirty/, "$proj dirty status is not dirty anymore");
     like($dirty_status, qr/published/, "$proj dirty is published");
 
     # click once again and make sure that timestamp on status changed
-    $driver->find_element("tr#folder_$proj .dirtystatusupdate")->click();
-    my $new_dirty_status = _wait_helper("tr#folder_$proj .dirtystatuscol .dirtystatus", sub { shift ne $dirty_status });
+    $driver->find_element("tr#folder_$ident .dirtystatusupdate")->click();
+    my $new_dirty_status
+      = _wait_helper("tr#folder_$ident .dirtystatuscol .dirtystatus", sub { shift ne $dirty_status });
```

Comment:
to make this wrap a bit nicer how about:

```perl
    my $dirtystatus_selector = 'tr#folder_$ident .dirtystatuscol .dirtystatus';
    my $new_dirty_status = _wait_helper($dirtystatus_selector, sub { shift ne $dirty_status });
```

Reply:
Please, I don't want to waste my, yours and CI time for this kind of improvements. Feel free to beautify code the way you want in separate PR if you think that priority for this is justified.

Reply:
That would be another PR with even more CI runs as at least there would be one run more on master. I can of course also do that change but for that I have to first wait for your PR to be merged, then create a branch of myself on top and change it which then someone else needs to review. I figured this approach doing it here is more efficient.

Reply:
My point is that you are free to do whatever you think worth your time, but don't think that others have free time to try all your ideas.

Reply:
@andrii-suse I don't think splitting this into 2 statements makes the code easier to read so it is questionable whether this is an improvement at all. Your version even has a little bit less characters and therefore looks a little bit "cleaner" to me. I don't know what @okurz has against the wrapping here. The `=` sign at the start of the 2nd line makes it super obvious and easy to follow.

Reply:
I don't really have any opinion about this and don't see how it can be worth discussing. But I don't mind if anybody changes it the way they prefer.

Reply:
I don't mind this too much, but prefer @okurz's version.

Reply:
so it's 2 vs. 2, so let's leave it :)